src/components/History/HistoryEntry.vue

```diff
@@ -48,8 +62,36 @@ export default {
       type: Object,
     },
   },
-  components: { ProfilePicture, DateAsWords, QItem, QItemSide, QItemMain, QItemTile },
+  data () {
+    return {
+      detailIsShown: false,
+    }
+  },
+  methods: {
+    toggleDetail (event) {
+      if (this.detailIsShown) {
+        window.history.replaceState({}, null, `#${this.$route.path}`)
+      }
+      else {
+        window.history.replaceState({}, null, `#/history/${this.entry.id}`)
```

Comment:
The component should remain agnostic of the exact route structure. You can (probably) achieve this with `this.$router.resolve({ name: 'historyDetail', params: { historyId: this.entry.id } }).href`.

Reply:
Oh, fancy! Makes sense :slightly_smiling_face: :+1: